src/convertToObject.js

```diff
@@ -3,16 +3,36 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const formattedCss = {};
+
+  sourceString
+    .split(';')
+    .forEach(rule => {
+      const [property, value] = rule.split(':');
+
+      if (property && value) {
+        const formattedProperty = removeSpaces(property);
```

Comment:
i think just trim() would also be ok in this case

Reply:
if it works, I agree =)

according to best practices we have 'to avoid making the code unnecessarily complicated'.
if you can use something simpler, then use it